server/README.md

```diff
@@ -98,6 +99,19 @@ cargo run
 
 Explore API available on `http://localhost:8000/graphql` with build in playground or try [online graphiql explorer](https://graphiql-online.com/)
 
+## Open mSupply Central and Remote
+
+Open mSupply server can be launched in both modes, this is controlled via `Site is open mSupply central server` configuration in legacy mSupply site settings, more info in [Sync Docs](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/README.md#open-msupply-central-server) and [Integration Test Docs](https://github.com/msupply-foundation/open-msupply/blob/develop/server/service/src/sync/test/integration/README.md#4-open-msupply-central-server).
+
+From version 2.0 omSupply would require both legacy and omSupply central server for full configuration/syncrhonisation. To start both omSupply central and omSupply remote:
```

Comment:
again - probably fine here, in internal docs: the preference for external documentation is to use 'Open mSupply' in preference to 'omSupply'

Reply:
Thanks, i'll try to change to use Open mSupply in internal docs too (and in some places I have), it's just so much easier to type omSupply